### DISCUSSION TITLE
Radar chart (spider chart)

### DISCUSSION THREAD

**Discussion by jsmariegaard** [DHI/modelskill#297] [Ideas]
The radar chart is becoming more and more popular.  I think it could be useful for multi-observation (at least 4) and multi-model cases. See the example below and imagine that "community 1" and "community 2" are two different models and each radii representing an observation. A single chart should show a single metric:   

![image](https://github.com/DHI/modelskill/assets/34088801/bc2428af-91eb-40fc-a622-e55009172caf)

Here is some sample matplotlib code that could get us started: 
https://matplotlib.org/stable/gallery/specialty_plots/radar_chart.html

**Comment by ecomodeller** [comment]
The ordering of cases (e.g. observation locations) suddenly becomes important for this type of visualization to make sense. If the visual is aimed at spotting some systematic patterns of model performance.

**Reply by jsmariegaard** [reply]
Yes, it is the same with plot.lines. Essentially this is just a polar version of plot.lines but with the advantage that you can mentally understand the performance of a model by assessing the covered area :-)

**Comment by jsmariegaard** [comment]
Recently, I have seen a lot of these when people compare performance of different LLMs:

![image](https://github.com/DHI/modelskill/assets/34088801/826b11bd-775a-4a0b-abd1-ae89721cb758)

**Comment by jsmariegaard** [comment]
@ovho-dhi - @clemenscremer tells me that you are an expert on this. I wonder if you could help us implement this or maybe add an example?

**Reply by ovho-dhi** [reply]
Sure, I'm happy to get involved! 

Are you planning this in the short term or generally for 2024? We currently have a project in which we are evaluating retention areas in the context of the water framework directive. The assessment is partly visualized with a radar plot. It was initially set up in R, but we are currently converting everything to Python. However, the project is currently on hold and I expect it to continue fundamentally towards Q2/Q3. Then I could combine the radar plotting for both, the project and modelskill.

**Reply by jsmariegaard** [reply]
Fantastic! Nothing urgent - Q2-Q3 sounds good. The 1.0 release will be out in a few weeks but we will keep developing ModelSkill this year
